index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.css" rel="stylesheet">
+  <link href="style.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:100,300,400" rel="stylesheet">
+</head>
+<body>
+  <div class="calculator">
+    <div class="calculator-answer-screen">0</div>
```

Comment:
This class name is specific, but when reading it I expect to see a 'calculator-answer-screen' div inside of a 'calculator-answer' div inside of a 'calculator' div. 

Just 'calculator-screen' or something similar would be specific enough for a div one level inside of the calculator container.